packages/sync-service/lib/electric/shapes/consumer.ex

```diff
@@ -434,12 +434,6 @@ defmodule Electric.Shapes.Consumer do
     now = DateTime.utc_now()
     lag = Kernel.max(0, DateTime.diff(now, commit_timestamp, :millisecond))
 
-    OpenTelemetry.with_span(
-      "shape_write.consumer.do_handle_txn.report_replication_lag",
-      [lag: lag],
-      fn ->
-        lag
-      end
-    )
+    OpenTelemetry.add_span_attributes(lag: lag)
```

Comment:
```suggestion
    OpenTelemetry.add_span_attributes(replication_lag: lag)
```

Attributes should be semantically self-contained as it gets confusing if we use `lag` somewhere else as then both would show up when you're searching in honeycomb.